pkg/config/config.go

```diff
@@ -0,0 +1,80 @@
+package config
+
+const (
+	InitConfigurationTemplate = `---
+{{.ClusterConfiguration}}
```

Comment:
templates are hard to test, it would be better to append these files to the same buffer